src/main/java/mate/controller/CreateDriverController.java

```diff
@@ -0,0 +1,36 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/create")
```

Comment:
check all your URL mapping with common mistakes 
<img width="493" alt="image" src="https://user-images.githubusercontent.com/61326568/191193115-2c3a2597-40ea-4820-98ca-5ead46c8d224.png">
```suggestion
@WebServlet(urlPatterns = "/drivers/add")
```